migration/migration-technical-details.md

```diff
@@ -66,14 +83,16 @@ During this phase, validators send a transaction signaling their readiness. Unli
 | Data      | Hash of the generated `GenesisConfig` |
 | Validity  | Activation block window               |
 
-An activation tool has been developed to facilitate the transition process. This tool scans the PoW chain for readiness transactions and selects the valid ones. Valid transactions come from pre-registered validators who have completed the registration process according to the abovementioned sequence. Additionally, the tool verifies the data field to ensure that the hash generated by the client matches the actual genesis config hash.
+#### Transition Block
+
+The `GenesisConfig` is generated after the candidate block is mined. If the pre-registered validators run the tool before the activation window starts, the tool will have already migrated most of the history by the time the candidate block is mined. After the candidate block is mined and the network reaches 80% readiness, the tool migrates the final part of the history and generates the "genesis" block, which becomes the transition block for the PoS chain.
 
-The activation tool activates at the candidate block - the block height aligned with an election block corresponding to the PoS chain’s structure. At this point, the tool scans and analyzes transactions sent to the burn address. The transition starts if at least 80% of the total stake is ready.
+The activation tool activates at the candidate block - the block height aligned with an election block corresponding to the PoS chain’s structure. At this point, the tool scans and analyzes transactions sent to the network. The transition starts if at least 80% of the total stake is ready.
```

Comment:
Remove " the block height aligned with an election block corresponding to the PoS chain’s structure"